pyth-sdk-terra/README.md

```diff
@@ -0,0 +1,86 @@
+# Pyth SDK Terra
+
+This crate provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle on the Terra network.
+The crate includes a library for reading and using Pyth data feeds in Terra.
+
+## Usage
+
+### Read price
+
+For reading the price you just need to call `query_price_feed` function within your contract with the id of the price.
+
+You can find the contract address and price ids in the section [Contracts and Price Feeds](#contracts-and-price-feeds) below.
+
+```rust
+let price_feed: PriceFeed = query_price_feed(deps.querier, contract_addr, price_id).unwrap().price_feed;
+```
+
+The `PriceFeed` struct contains several useful functions for working with the price.
+Some of these functions are described below.
+For more detailed information, please see the crate documentation.
+
+
+### Get the current price
+
+Read the current price from a `PriceFeed`: 
+
+```rust
+let current_price: Price = price.get_current_price().unwrap();
+println!("price: ({} +- {}) x 10^{}", current_price.price, current_price.conf, current_price.expo);
+```
+
+The price is returned along with a confidence interval that represents the degree of uncertainty in the price.
```

Comment:
It might be nice to link to some documentation on the pyth.network website which explains a little more about the confidence interval.

Reply:
Maybe also add a warning or something, highlighting the existence of the confidence interval?

Reply:
Added an warning to best practices as you suggested.